templates/zerver/help/import-from-rocketchat.md

```diff
@@ -1,98 +1,127 @@
+
+{end_tabs}
+
+## Import your organization from Rocket.Chat into Zulip
+
+### Export your Rocket.Chat data
+
+Rocket.Chat does not provide an official data export feature, so the Zulip
+import tool works by importing data from a Rocket.Chat database dump.
 
 If you're self-hosting your Rocket.Chat instance, you can create a
 database dump using the `mongodump` utility.
 
 If your organization is hosted on Rocket.Chat Cloud or another hosting
 provider that doesn't provide you with database access, you will need
-to request a database dump by contacting their support.
+to request a database dump by contacting their
+[support](https://docs.rocket.chat/resources/frequently-asked-questions/cloud-faqs#data-export).
 
 In either case, you should end up with a directory containing many
 `.bson` files.
 
-### Import into Zulip Cloud
+### Import your data into Zulip
+
+{!import-your-data-into-zulip.md!}
+
+At this point, you should go to the directory containing all the `.bson` files
+from your database dump and rename it to `rocketchat_data`. This directory will
+be your **exported data** file in the instructions below.
+
+{start_tabs}
+
+{tab|zulip-cloud}
 
-Email support@zulip.com with your database dump and your desired
-Zulip subdomain. Your imported organization will be hosted at
-`<subdomain>.zulipchat.com`.
+{!import-into-a-zulip-cloud-organization.md!}
 
-If you've already created a test organization at
-`<subdomain>.zulipchat.com`, let us know, and we can rename the old
-organization first.
+{!import-zulip-cloud-organization-warning.md!}
 
-### Import into a self-hosted Zulip server
+{tab|self-hosting}
 
-First [install a new Zulip
-server](https://zulip.readthedocs.io/en/stable/production/install.html)
-with Zulip 5.0 or newer, skipping "Step 3: Create a Zulip
-organization, and log in" (you'll create your Zulip organization via
-the data import tool instead).
+{!import-into-a-self-hosted-zulip-server.md!}
 
-Now, get the directory containing all the `bson` files in your database
-dump and save it inside `/home/zulip/rocketchat` on your Zulip server and rename it
-to `rocketchat` (the directory at `/home/zulip/rocketchat` should contain
-all the `bson` files).
+1. To import into an organization hosted on the root domain
+   (`EXTERNAL_HOST`) of the Zulip installation, run the following
+   commands.
 
-Log in to a shell on your Zulip server as the `zulip` user. To import with
-the most common configuration, run the following commands:
+{!import-self-hosted-server-tips.md!}
 
-```
-cd /home/zulip/deployments/current
-./scripts/stop-server
-./manage.py convert_rocketchat_data /home/zulip/rocketchat --output /home/zulip/converted_rocketchat_data
-./manage.py import "" /home/zulip/converted_rocketchat_data
-./scripts/start-server
-```
+    ```
+    cd /home/zulip/deployments/current
+    ./scripts/stop-server
+    ./manage.py convert_rocketchat_data /tmp/rocketchat_data --token <token> --output /tmp/converted_rocketchat_data
```

Comment:
I think the `token` arg is a mistake from copying over the Slack instructions? We don't have a token involved in the rocket chat import right?

Reply:
Correct. Good catch!